File: leetcode/day8/cpp/main.cpp
#include <vector>
#include <iostream>
#include "so.h"

int main()
{
    Solution so{};

    std::vector<int> a = {2, 3, 1, 0, 2, 5, 3};
    // std::cout << so.findRepeatNumber(a) << std::endl;

    a = {3, 4, 2, 1, 1, 0};
    std::cout << so.findRepeatNumber(a) << std::endl;

    return 0;
}

File: leetcode/day8/python/main.py

import so
from typing import List


def main():
    s = so.Solution()
    a: List[int] = [3, 4, 2, 1, 1, 0]
    # print(a[0])
    print("res: {}".format(s.findRepeatNumber(a)))


if __name__ == "__main__":
    main()
